openapi.yml

```diff
@@ -3095,6 +3102,7 @@ paths:
             - affects__type
             - affects__updated_dt
             - affects__uuid
+            - bz_id
```

Comment:
I am either hallucinating or I see some items twice in the list :eyes: can you please double check it?

Reply:
There is `-bz-id` and `bz-id` for descending and ascending order.

Reply:
it adds all items twice one for asc and one for desc like: `bz_id` and `-bz_id` in this case.

Reply:
I see! The difference is not too visible for a tired human :smile: